deployment/terraform/iam.tf

```diff
@@ -0,0 +1,82 @@
+#
+# Container Instance IAM resources
+#
+data "aws_iam_policy_document" "container_instance_ec2_assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+resource "aws_iam_role" "container_instance_ec2" {
```

Comment:
It may be valuable to proactively attach an IAM policy that allows read/write to S3 to this role. Similarly, it may be valuable to create a project specific data bucket in Terraform and scope those read/write operations to it.